CHANGELOG.md

```diff
@@ -0,0 +1,178 @@
+
+**New features:**
+
+- Allow k-DOP as a bounding volume in BVH [\#386](https://github.com/arborx/ArborX/pull/386)
+
+**Enhancements:**
+
+- Store a dimension value in a DBSCAN datafile [\#523](https://github.com/arborx/ArborX/pull/523)
+- Provide computation of a Sphere-Box distance [\#495](https://github.com/arborx/ArborX/pull/495)
+- Provide Box-Box distance for the Nearest query [\#494](https://github.com/arborx/ArborX/pull/494)
+- Use double to normalize Ray direction [\#535](https://github.com/arborx/ArborX/pull/535)
+- Improve performance of the brute force algorithm [\#534](https://github.com/arborx/ArborX/pull/534)
+- Improve performance of the DBSCAN algorithm (new FDBSCAN-DenseBox algorithm) [\#508](https://github.com/arborx/ArborX/pull/508)
+- Allow non-View input for DBSCAN [\#509](https://github.com/arborx/ArborX/pull/509)
+- Install ArborXConfigVersion.cmake [\#521](https://github.com/arborx/ArborX/pull/521)
+- Add a simple intersection example [\#513](https://github.com/arborx/ArborX/pull/513)
+- Improve performance for the SYCL backend through the use of oneDPL for sorting [\#456](https://github.com/arborx/ArborX/pull/456)
+
+**Fixed bugs:**
+
+- Fix a bug in DBSCAN noise marking [\#525](https://github.com/arborx/ArborX/pull/525)
+
+## [v1.0](https://github.com/arborx/arborx/tree/v1.0) (2021-03-13)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.0-rc0...v1.0)
+
+**New features:**
+
+- Allow early termination of a traversal by a thread [\#427](https://github.com/arborx/ArborX/pull/427)
+- Implement DBSCAN clustering algorithm [\#331](https://github.com/arborx/ArborX/pull/331)
+- Implement brute-force algorithm [\#468](https://github.com/arborx/ArborX/pull/468)
+- Implement simple ray-tracing [\#414](https://github.com/arborx/ArborX/pull/414)
+
+**Build changes:**
+
+- Require CMake 3.16 [\#486](https://github.com/arborx/ArborX/pull/486)
+
+**Enhancements:**
+
+- Add KOKKOS\_FUNCTION to AccessTraits::size\(\) in View specialization [\#463](https://github.com/arborx/ArborX/pull/463)
+- Allow running BVH benchmark with SYCL and OpenMPTarget explicitly [\#455](https://github.com/arborx/ArborX/pull/455)
+- Change signature of the nearest callback [\#366](https://github.com/arborx/ArborX/pull/366)
+- Add a free function constructing CRS graph [\#425](https://github.com/arborx/ArborX/pull/425)
+- Improve performance for the HIP backend through the use of rocThrust for sorting [\#424](https://github.com/arborx/ArborX/pull/424)
+- Add CI for SYCL [\#423](https://github.com/arborx/ArborX/pull/423)
+
+## [v1.0-rc0](https://github.com/arborx/arborx/tree/v1.0-rc0) (2020-10-03)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v0.9-beta...v1.0-rc0)
+
+**New features:**
+
+- New BVH::query\(\) overload that only takes predicates and callback [\#329](https://github.com/arborx/ArborX/pull/329)
+
+**Implemented enhancements:**
+
+- Change naming of ArborX labels [\#362](https://github.com/arborx/ArborX/issues/362)
```

Comment:
Combine with 380 and say "ensure all kernels and memory allocation are prefixed with `ArborX::`"